### PR TITLE
341629 Add new attributes (width/depth/manufacturer) to Bay

### PR DESCRIPTION
**Pull request by B-Rass**

### CODE REVIEW

File: app/views/manufacturers/index.html.erb [hunk 1]
```diff
@@ -21,10 +21,14 @@
       <%= link_to manufacturer, manufacturer_path(manufacturer) %>
     <% end %>
 
-    <% table.with_column(Manufacturer.human_attribute_name(:usage), sort_by: :modeles_count) do |manufacturer| %>
+    <% table.with_column(Manufacturer.human_attribute_name(:usage_modele), sort_by: :modeles_count) do |manufacturer| %>
       <%= Manufacturer.human_attribute_name(:modeles_count, count: manufacturer.modeles_count) %>
     <% end %>
 
+    <% table.with_column(Manufacturer.human_attribute_name(:usage_bay), sort_by: :bays_count) do |manufacturer| %>
+      <%= Manufacturer.human_attribute_name(:bays_count, count: manufacturer.bays_count) %>
```
Comment (nicolas-brousse) [inline app/views/manufacturers/index.html.erb:29]: Maybe we could add like to filtered bays list view

File: app/views/manufacturers/index.html.erb [hunk 2]
```diff
@@ -21,10 +21,14 @@
       <%= link_to manufacturer, manufacturer_path(manufacturer) %>
     <% end %>
 
-    <% table.with_column(Manufacturer.human_attribute_name(:usage), sort_by: :modeles_count) do |manufacturer| %>
+    <% table.with_column(Manufacturer.human_attribute_name(:usage_modele), sort_by: :modeles_count) do |manufacturer| %>
       <%= Manufacturer.human_attribute_name(:modeles_count, count: manufacturer.modeles_count) %>
```
Comment (nicolas-brousse) [inline app/views/manufacturers/index.html.erb:25]: Maybe we could add like to filtered modeles list view

File: app/views/bays/show.html.erb [hunk 3]
```diff
@@ -22,6 +22,17 @@
...
+          <dt class="pb-2"><%= Manufacturer.model_name.human %></dt>
+          <dd class="mb-0 pb-2 ps-3">
+            <%= link_to @bay.manufacturer, manufacturer_path(@bay.manufacturer) if @bay.manufacturer.present? %>
+          </dd>
+
+          <dt class="pb-2"><%= Bay.human_attribute_name(:width) %></dt>
+          <dd class="mb-0 pb-2 ps-3"><%= value_with_unit(@bay.width, "mm") if @bay.width.present? %></dd>
+
+          <dt class="pb-2"><%= Bay.human_attribute_name(:depth) %></dt>
+          <dd class="mb-0 pb-2 ps-3"><%= value_with_unit(@bay.depth, "mm") if @bay.depth.present? %></dd>
```
Comment (nicolas-brousse) [inline app/views/bays/show.html.erb:35]: What about updating `value_with_unit` to not print the unit when value is blank?

File: app/processors/bays_processor.rb [hunk 4]
```diff
@@ -17,5 +17,9 @@ class BaysProcessor < ApplicationProcessor
     raw.where(islet_id: islet_id)
   end
 
+  map :manufacturer_id do |manufacturer_id:|
+    raw.where(manufacturer_id: manufacturer_id)
+  end
+
```
Comment (nicolas-brousse) [inline app/processors/bays_processor.rb:23]: Seems test to be missing

**Review by nicolas-brousse — CHANGES_REQUESTED** [review]

File: app/views/manufacturers/index.html.erb [hunk 5]
```diff
@@ -21,8 +21,16 @@
...
+    <% table.with_column(Manufacturer.human_attribute_name(:usage_modele), sort_by: :modeles_count) do |manufacturer| %>
+      <%= link_to modeles_path(manufacturer_id: manufacturer) do %>
+        <%= Manufacturer.human_attribute_name(:modeles_count, count: manufacturer.modeles_count) %>
+      <% end %>
+    <% end %>
+
+    <% table.with_column(Manufacturer.human_attribute_name(:usage_bay), sort_by: :bays_count) do |manufacturer| %>
+      <%= link_to bays_path(manufacturer_id: manufacturer) do %>
+        <%= Manufacturer.human_attribute_name(:bays_count, count: manufacturer.bays_count) %>
+      <% end %>
```
Comment (nicolas-brousse) [inline app/views/manufacturers/index.html.erb:33]: May we add the filter in filter form in sites too?

**Review by nicolas-brousse — COMMENTED** [review]
@B-Rass one last question

File: app/views/manufacturers/index.html.erb (hunk 5 above)
Comment (B-Rass) [reply app/views/manufacturers/index.html.erb:33]: Yes you're right, done

**Pull request merged**